services/ui-waf-log-assoc/serverless.yml

```diff
@@ -22,7 +22,6 @@ custom:
       - impl
       - val
       - prod
-  iamPath: ${env:IAM_PATH, "/"}
```

Comment:
This was just something tha is very OBE that I saw.  Although non functional, since it was in the realm of iam path (that's why I found it), I just removed it.